instrumentation/README.md

```diff
@@ -10,7 +10,7 @@
 | [opentelemetry-instrumentation-boto](./opentelemetry-instrumentation-boto) | boto~=2.0 | No
 | [opentelemetry-instrumentation-boto3sqs](./opentelemetry-instrumentation-boto3sqs) | boto3 ~= 1.0 | No
 | [opentelemetry-instrumentation-botocore](./opentelemetry-instrumentation-botocore) | botocore ~= 1.0 | No
-| [opentelemetry-instrumentation-celery](./opentelemetry-instrumentation-celery) | celery >= 4.0, < 6.0 | No
+| [opentelemetry-instrumentation-celery](./opentelemetry-instrumentation-celery) | celery >= 4.0, < 6.0 | Yes
```

Comment:
@srikanthccv 

Any suggestions on how to expose this to users that the type of metrics collected is seen in [this](https://flower.readthedocs.io/en/latest/prometheus-integration.html#available-metrics) list instead of opentelemetry defined task-metrics?

Reply:
I am not sure about this. Can we eliminate the `flower.` prefix, keep the rest of the name, and also include some other messaging metrics taking inspiration from Kafka https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions/instrumentation?

Reply:
I'm fine with doing those things. Perhaps we also shouldn't mark "yes" for metrics being supported as my understanding for that list is support for OpenTelemetry defined metrics?

Reply:
Yes, we only want to mart "yes" if it follows the spec conventions.

Reply:
Changed

Reply:
Changed